gpkit/small_scripts.py

```diff
@@ -37,7 +42,8 @@ def unitstr(units, into="%s", options="~", dimless='-'):
         except ValueError:
             rawstr = "1.0 " + str(units.units)
         units = "".join(rawstr.replace("dimensionless", dimless).split()[1:])
-    return into % units if units else ""
+        return into % units if units else ""
+    return ""
```

Comment:
to make logistics simple I'll open a separate PR from that branch for further discussion